examples/with-remix-thirdpartyemailpassword/app/routes/api.auth.$.tsx

```diff
@@ -0,0 +1,82 @@
+import { json } from "@remix-run/node";
+import { ActionFunctionArgs, LoaderFunctionArgs } from "react-router-dom";
+import { getAppDirRequestHandler } from "supertokens-node/nextjs/index.js";
```

Comment:
should not use nextjs, cause this is not a nextjs app. it's a remix app. Please figure out a way to get it to work with the custom framework itself.

Reply:
Since this function is used by both Remix and NextJS there isn't a reason to create a separate function for Remix. May I recommend the team make `getAppDirRequestHandler` a general function that can be called by either framework? It isn't practical to recreate this functionality for Remix.

Let me know.

Reply:
See my draft PR for a suggestion: [https://github.com/supertokens/supertokens-node/pull/798](https://github.com/supertokens/supertokens-node/pull/798)

Reply:
We can repicate the logic in the docs. It's not a big deal for now

Reply:
I brought the function down into the project and customized it for Remix.